app/Repositories/PostRepository.php

```diff
@@ -42,7 +43,10 @@ public function __construct(AWS $aws, Registrar $registrar)
     public function create(array $data, $signupId)
     {
         if (isset($data['file'])) {
-            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+            // Auto-orient the photo by default based on exif data.
+            $img = Image::make($data['file'])->orientate();
```

Comment:
Totally nitpicky, but I think in general we prefer spelling out full variable names: `$image` over `$img`.